MuvrKit/MKPhoneExerciseSessionClassifier.swift

```diff
@@ -46,9 +46,31 @@ public final class MKSessionClassifier : MKExerciseConnectivitySessionDelegate,
     }
     
     private func classify(exerciseModelId exerciseModelId: MKExerciseModelId, sensorData: MKSensorData) -> [MKClassifiedExercise]? {
-        let model = exerciseModelSource.getExerciseModel(id: exerciseModelId)
-        let classifier = try! MKClassifier(model: model)
-        return try? classifier.classify(block: sensorData, maxResults: 10)
+        do {
+            let slackingModel = exerciseModelSource.getExerciseModel(id: "slacking")
```

Comment:
We can probably cache the ``slackingModel``; even better, load it once the ``exerciseModelSource`` is set.